cmd/kaf/consume.go

```diff
@@ -46,6 +47,7 @@ func init() {
 	consumeCmd.Flags().StringVar(&protoType, "proto-type", "", "Fully qualified name of the proto message type. Example: com.test.SampleMessage")
 	consumeCmd.Flags().StringVar(&keyProtoType, "key-proto-type", "", "Fully qualified name of the proto key type. Example: com.test.SampleMessage")
 	consumeCmd.Flags().Int32SliceVarP(&flagPartitions, "partitions", "p", []int32{}, "Partitions to consume from")
+	consumeCmd.Flags().Int64VarP(&limit, "limit", "l", 0, "Limit messages per partition")
```

Comment:
my opinion is, that `limit` does not sound obvious enough. if i read `limit`, i don't immediately know what the limit applies to.

one option would making the flag more complicated, `limit-messages`, `num-messages`, `messages-per-partitions`. not sure..i have to think about it. what's your opinion here?

Reply:
`limit-messages`   sounds good to me